### PR TITLE
Added token attribute with max value of 3

### LINKED ISSUE #221: Editing the user model to incorporate tournament tokens
In this issue, we will edit the user model to add an attribute for tournament tokens. All users start with one token and will eventually automatically get a new token every 30 days. The maximum token balance will be 3.

### PR DESCRIPTION
**Pull request by 24raniwalar**
We have edited the user model to add a token attribute that takes a default value of 1 and has a max value of 3. Any time it exceeds this max value it is reset to 3.

### CODE REVIEW

File: src/chigame/users/models.py [hunk 1]
```diff
@@ -37,6 +39,12 @@ def get_absolute_url(self) -> str:
         """
         return reverse("users:detail", kwargs={"pk": self.id})
 
+    def save(self, *args, **kwargs):
+        if self.tokens > 100:
```
Comment (MuyanXie) [inline src/chigame/users/models.py]: double check this line. Maybe it should be 3 instead of 100?

**Review by MuyanXie — COMMENTED** [review]
This looks good to me! However, I have one small concern about this. I noticed on line 43-44 in models.py, the maximum amount of token is 100 instead of 3.

**Review by MuyanXie — APPROVED** [review]
Great Job! Approved

**Review by elizabethli31 — CHANGES_REQUESTED** [review]
Migrations and model changes are solid, but you built this branch of users/inbox so it's including all changes you made on that branch. Please remove those changes and make sure this branch matches dev. This PR should only have changes in migrations and models.

**Review by elizabethli31 — APPROVED** [review]
LGTM! Rebase and I'll merge

**Comment by elizabethli31** [comment]
Issue Score: Satisfactory

Comments:
Good minor change. Issues and PRs should only be labeled with one team. I've removed tournaments, but next time, be sure to only label your own team. Also, missing closing statement.

**Pull request merged**